go/processor/geth/processor_test.go

```diff
@@ -94,3 +97,123 @@ func TestGethProcessor_ConfigAddsStateContract(t *testing.T) {
 		t.Errorf("state contract not added to config")
 	}
 }
+
+func TestTransactionToMessage_BasicFields(t *testing.T) {
+	tx := tosca.Transaction{
+		Sender:        tosca.Address{0x01},
+		Recipient:     &tosca.Address{0x02},
+		Nonce:         42,
+		Value:         tosca.NewValue(1000),
+		GasLimit:      21000,
+		GasFeeCap:     tosca.NewValue(50),
+		GasTipCap:     tosca.NewValue(2),
+		Input:         []byte{0x42, 0x42},
+		BlobGasFeeCap: tosca.NewValue(0),
+	}
+	gasPrice := tosca.NewValue(50)
```

Comment:
This price should be something different than the `GasFeeCap` to make sure the `GasPrice` in the result is taken from this parameter and not from the `GasFeeCap`.